### PR TITLE
Update de-DE.hjson

### PR DESCRIPTION
**Pull request by 3incognito**
I have tried this mod and it works. So I thought I would add the missing German translation.

Note:
The English terms 'multiplayer' and 'singleplayer' have become firmly established in the German gaming community and are generally accepted, which is why I am also using them here. The literal German translations would be 'Mehrspieler' for 'multiplayer' and 'Einzelspieler' for 'singleplayer', but these are rarely used in the everyday language of gamers.

### CODE REVIEW

File: Localization/de-DE.hjson [hunk 1]
```diff
@@ -2,30 +2,30 @@ Mods: {
...
+					Tooltip: "{$Configs.Config.Brightness.Tooltip}"
 				}
 
 				UseServerSettings: {
-					// Label: Use Server Settings
-					/* Tooltip:
+					Label: Verwende Server Einstellungen
+					Tooltip:
+						'''
+						Wenn aktiviert, überschreibt '{$Configs.Config.Brightness.Label}' vom Server (via '{$Configs.Config.DisplayName}') diese Einstellung
```
Comment (direwolf420) [inline Localization/de-DE.hjson:26]: `Wenn aktiviert, wird '{$Configs.Config.Brightness.Label}' vom Server (via '{$Configs.Config.DisplayName}') überschrieben`

**Review by direwolf420 — COMMENTED** [review]
Thanks, I suggested a small change to make it sound better. Also, if you are interested, I'm giving out steam gifts for enough contributed localizations: https://gist.github.com/direwolf420/ad3d2a7785764bd26f81284007968082

File: Localization/de-DE.hjson (hunk 1 above)
Comment (3incognito) [reply Localization/de-DE.hjson:26]: Yes, that sounds better.👍

**Comment by direwolf420** [comment]
I guess I'll merge it and add my suggestion, then publish. What name do you want to be credited as?

**Comment by 3incognito** [comment]
Sure, you can credit me with my GitHub name.

**Pull request merged**